gcc 11.2.0でコンパイルエラー

ubuntu 22.04 gcc (Ubuntu 11.2.0-19ubuntu1) 11.2.0
の環境でビルドしたところ以下のエラーが発生しました。
エラーの詳細原因は不明ですが、mros_comm_socket_select()のmRosFdSetType fd_set;の変数名がfd_setになっていることが原因のようです。fd_setをsetのような名前に変更することでビルドが通ることを確認しました。
```
In file included from /usr/include/sys/select.h:30,
                 from /usr/include/sys/types.h:179,
                 from /usr/include/stdlib.h:395,
                 from ../../athrill/src/device/peripheral/mros-dev/mros-athrill/target/os/kernel.h:5,
                 from ../../athrill/src/device/peripheral/mros-dev/mros-src/os/target/os_asp/mros_os_target.h:8,
                 from ../../athrill/src/device/peripheral/mros-dev/mros-src/inc/mros_types.h:51,
                 from ../../athrill/src/device/peripheral/mros-dev/mros-src/comm/cimpl/mros_comm_cimpl.h:25,
                 from ../../athrill/src/device/peripheral/mros-dev/mros-src/comm/cimpl/mros_comm_socket_cimpl.h:25,
                 from ../../athrill/src/device/peripheral/mros-dev/mros-src/comm/cimpl/mros_comm_socket_cimpl.c:18:
../../athrill/src/device/peripheral/mros-dev/mros-src/comm/cimpl/mros_comm_socket_cimpl.c: In function ‘mros_comm_socket_select’:
../../athrill/src/device/peripheral/mros-dev/mros-src/comm/cimpl/target/lwip/mros_comm_target.h:37:57: error: ‘__arr’ undeclared (first use in this function)
   37 | #define MROS_FD_ZERO(arg)                               FD_ZERO(arg)
      |                                                         ^~~~~~~
../../athrill/src/device/peripheral/mros-dev/mros-src/comm/cimpl/mros_comm_socket_cimpl.c:67:9: note: in expansion of macro ‘MROS_FD_ZERO’
   67 |         MROS_FD_ZERO(&fd_set);
      |         ^~~~~~~~~~~~
../../athrill/src/device/peripheral/mros-dev/mros-src/node/cimpl/mros_node_cimpl.c
../../athrill/src/device/peripheral/mros-dev/mros-src/comm/cimpl/target/lwip/mros_comm_target.h:37:57: note: each undeclared identifier is reported only once for each function it appears in
   37 | #define MROS_FD_ZERO(arg)                               FD_ZERO(arg)
      |                                                         ^~~~~~~
../../athrill/src/device/peripheral/mros-dev/mros-src/comm/cimpl/mros_comm_socket_cimpl.c:67:9: note: in expansion of macro ‘MROS_FD_ZERO’
   67 |         MROS_FD_ZERO(&fd_set);
      |
```

--- a/src/device/peripheral/mros-dev/mros-src/comm/cimpl/mros_comm_socket_cimpl.c
+++ b/src/device/peripheral/mros-dev/mros-src/comm/cimpl/mros_comm_socket_cimpl.c
@@ -61,20 +61,20 @@ void mros_comm_socket_close(mRosCommSocketType *socket)
 
 static mRosReturnType mros_comm_socket_select(mRosCommSocketType *socket, mros_uint32 timeout, mros_boolean read, mros_boolean write) {
 
-	mRosFdSetType fd_set;
+	mRosFdSetType set;
 	mRosTimeValType tmo;
 
-	MROS_FD_ZERO(&fd_set);
-    MROS_FD_SET(socket->sock_fd, &fd_set);
+	MROS_FD_ZERO(&set);
+    MROS_FD_SET(socket->sock_fd, &set);
 
     mRosFdSetType* r_set = MROS_NULL;
     mRosFdSetType* w_set = MROS_NULL;
 
     if (read == MROS_TRUE) {
-    	r_set = &fd_set;
+    	r_set = &set;
     }
     if (write == MROS_TRUE) {
-    	w_set = &fd_set;
+    	w_set = &set;
     }
     mros_comm_set_timeval(timeout, &tmo);
     mRosReturnType ret = mros_comm_select(MROS_FD_SETSIZE, r_set, w_set, MROS_NULL, &tmo);
@@ -82,7 +82,7 @@ static mRosReturnType mros_comm_socket_select(mRosCommSocketType *socket, mros_u
 		ROS_ERROR("%s %s() %u ret=%d", __FILE__, __FUNCTION__, __LINE__, ret);
     	return MROS_E_SYSERR;
     }
-    if ((ret == 0 || !MROS_FD_ISSET(socket->sock_fd, &fd_set))) {
+    if ((ret == 0 || !MROS_FD_ISSET(socket->sock_fd, &set))) {
     	return MROS_E_NOENT;
     }
     return MROS_E_OK;

